src/background-plugin.ts

```diff
@@ -37,7 +32,7 @@ const AtomicsWaitAsync =
 
     const blob = new (Blob as any)([src], { type: 'text/javascript' });
     const url = URL.createObjectURL(blob);
-    const w = new Worker(url);
+    const w = new Worker(url, { execArgv: [] });
```

Comment:
This worker is a fallback for `Atomic.waitAsync` being unavailable. Always null out the `execArgv`.